### ISSUE TITLE
Download for pretrained generators do not seem to work.

### ISSUE THREAD

**Issue by amonsoes** [ShawnXYang/C-GSP#1]
Hello, thank you for releasing your code. Could you provide the links to download the pretrained generators for evaluation on the Nips17 ImageNet validation set again? The ones featured in the README file do not seem to work.

Thank you!
